modules/create-wif-cluster-cli.adoc

```diff
@@ -205,18 +221,31 @@ ocm gcp update wif-config --version <version> \ <1>
 <1> Replace `<version>` with the {product-title} y-stream version you plan to update the cluster to.
 <2> Replace `<wif_name>` with the name of the WIF configuration you want to update.
 
-[id="ocm-cli-list-wif-commands_{context}"]
-== List WIF clusters
+[id="ocm-cli-verify-wif-commands_{context}"]
+== Verifying a WIF configuration
+You can verify that the configuration of resources associated with a wif-config are correct by running the `ocm gcp verify wif-config` command. If a misconfiguration is found, output will identify the specific misconfiguration and recommend that you update the WIF configuration.
```

Comment:
```suggestion
You can verify that the configuration of resources associated with a wif-config are correct by running the `ocm gcp verify wif-config` command. The output identifies the specific misconfiguration and recommends that you update the WIF configuration.
```

- Consider using the present tense as per the following guideline:
https://redhat-documentation.github.io/supplementary-style-guide/#will
> Description: Do not use future tense unless it is absolutely necessary.